theano/d3viz/__init__.py

```diff
@@ -2,8 +2,10 @@
 
 has_requirements = False
 try:
-    import pydot as pd
-    if pd.find_graphviz():
-        has_requirements = True
+    # pydot-ng is a fork of pydot that is better maintained
+    import pydot_ng as pd
 except ImportError:
-    pass
+    # fall back on pydot if necessary
+    import pydot as pd
+if pd.find_graphviz():
+    has_requirements = True
```

Comment:
while cleaning this, could you rename has_requirement as the other pydot_imported for consistency?